src/Entity/Aid.php

```diff
@@ -0,0 +1,464 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\FundraisingCardRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+use Symfony\Bridge\Doctrine\IdGenerator\UuidV4Generator;
+use Symfony\Component\Uid\UuidV4;
+
+/**
+ * @Gedmo\SoftDeleteable(fieldName="deletedAt", timeAware=false, hardDelete=false)
+ * @ORM\Entity(repositoryClass=FundraisingCardRepository::class)
+ * @ORM\Table(name="aids")
+ */
+class Aid
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="uuid", unique=true)
+     * @ORM\GeneratedValue(strategy="CUSTOM")
+     * @ORM\CustomIdGenerator(class=UuidV4Generator::class)
+     */
+    private UuidV4 $id;
```

Comment:
Not a good idea: we should use autoincremented numbers as ID and use UUID for URLs/public usages (or even better, ULID). See https://titouangalopin.com/auto-increment-is-the-devil-using-uuids-in-symfony-and-doctrine/